VideoLocker/src/main/java/org/edx/mobile/util/images/ErrorUtils.java

```diff
@@ -16,31 +17,43 @@
 
     protected static final Logger logger = new Logger(ErrorUtils.class.getName());
 
+    public static class Error {
+        public MessageType type;
+        public String message;
+    }
```

Comment:
Might as well just take the `TaskMessageCallback` as the parameter, and invoke it from here in this case. I don't think there is any case where we would need to just query the appropriate error without actually posting it.